index.rst

```diff
@@ -1,13 +1,15 @@
 Welcome to the UBports documentation!
 =====================================
 
-Welcome to the official documentation of the UBports project!
+Welcome to the official documentation of the UBports project! We are a team of engineers, designers, community managers, translators, students and volunteers who together bring to life Ubuntu Touch, a mobile operating system focused on ease of use, privacy, and convergence. We are also lovers of Free software and open collaboration. You can find us all over the web, but Github provides the best starting point for planning your involvement. If you are new, head to get.ubuntu.com to get started
```

Comment:
* get.ubuntu.com doesn't go anywhere
* I'm not sure that Github is the best starting point for planning involvement. I think some social outlet would serve better: e.g.,  forum, https://ubports.com/join-us, or https://ubports.com/contact
* and since i'm already nitpicking if you capitalize "Free" (which I like) we should also capitalize "Software" and the last sentence doesn't end with a period.

Reply:
https://ubuntu-touch.io/get-ubuntu-touch is the link we are after, right?

I don't know what happens to people registering under the join-us page. But my impression is: nothing. So I would not link to that page. The contacts page is good. Also the hidden page [https://ubports.com/meet-the-community](https://ubports.com/meet-the-community) might be good here.

I agree that github is not the best entry point. For this doc maybe, but currently efforts are moving everything over to gitlab. So if we want to link there, the [epics page](https://gitlab.com/groups/ubports/-/epics) would be possible. But again maybe not the right spot for this.

Reply:
Dab said it right. I'm sorry about the link AI came with this ;)